src/cargo/util/toml/mod.rs

```diff
@@ -1313,9 +1318,24 @@ impl TomlManifest {
             }
         }
 
+        let default_kind = project
+            .default_target
+            .as_ref()
+            .map(|t| CompileTarget::new(&*t))
+            .transpose()? // TODO: anyhow::Context isn't imported yet so I guess .context() isn't the right way to do it?
```

Comment:
I think it's safe to remove this `TODO`?

Reply:
Hmm, it's true that I had totally forgotten about this TODO, sorry! It was there because I was thinking that there may be a want to add details to the error message, but I have basically no knowledge of the cargo error handling story yet.

I've removed this TODO for now, feel free to let me know if you think there should be an equivalent to `.context()` here to give a hint to the user that the issue comes from a mis-spelled `default_target` (if I understood correctly the error conditions of `CompileTarget::new`) :)